plugins/services/src/js/structs/PodSpec.js

```diff
@@ -77,6 +77,9 @@ module.exports = class PodSpec extends ServiceSpec {
   getVolumes() {
     return this.get("volumes") || [];
   }
+  getVolumesData() {
+    return this.get("volumeData") || [];
```

Comment:
I feel unsafe having the same name that in one case returns an array and in another returns a `List`

Reply:
☑️